server/translations.ts

```diff
@@ -141,22 +142,45 @@ function getDocument(filePath) {
     };
   }
 
+  function getCommitBehindFromLatest(filename, commitHash) {
+    const commitHashes = execSync(
+      `cd ${CONTENT_ROOT} && git log --pretty=format:%H -- ${filename}`
```

Comment:
I get the index of the file's latest hash list like this and see how far behind it is. However, there are significant delays. I'm not sure, but it seems that source-commit is slowing down by about half a second per file written.

For the PoC, I wrote it as it is at first, but we need another method. Or do we need pre-processing to write to a separate file for count?